packages/components/src/components/teaser-media/teaser-media.stories.ts

```diff
@@ -0,0 +1,257 @@
+        },
+        {
+          type: 'slot',
+          name: 'media',
+          value:
+            '<img slot="media" class="aspect-video object-cover" src="./placeholders/generic.jpg" alt="A skyline of a city by night"/>'
+        }
+      ]
+    });
+  }
+};
+
+/**
+ * Use the 'default', 'media', 'meta', 'expandable' and 'headline' slots to add content to the teaser. Please use h1-h6 tags for the headline slot.
+ */
+export const Slots = {
+  parameters: {
+    controls: { exclude: ['default', 'media', 'meta', 'headline', 'expandable'] }
+  },
+  render: (args: any) => {
+    return html`
+      ${['default', 'media', 'meta', 'headline', 'expandable'].map(slot => {
+        let value = `<div slot='${slot}' class="slot slot--border slot--background h-16"></div>`;
+
+        if (slot === 'default') {
+          value = `<div class="slot slot--border slot--background h-16"></div>`;
+        } else if (slot === 'media') {
+          value = `<div slot='${slot}' class="slot slot--border slot--background h-[577px]"></div>`;
+        }
+
+        return generateTemplate({
+          axis: {
+            x: {
+              type: 'slot',
+              name: slot,
+              title: 'slot=..',
+              values: [
+                {
+                  value: value,
+                  title: slot
+                }
+              ]
+            }
+          },
+          args,
+          constants: [
+            {
+              type: 'slot',
+              name: 'media',
+              value: `<img slot='media' src='./placeholders/collaboration.jpg' alt='Test'/>`
+            },
+            {
+              type: 'slot',
+              name: 'meta',
+              value: `<slot slot='meta'>Teaser's Meta information</slot>`
+            },
+            {
+              type: 'slot',
+              name: 'default',
+              value: `<slot>Teaser's Main content</slot>`
+            },
+            {
+              type: 'slot',
+              name: 'headline',
+              value: `<slot slot='headline'>Teaser's Headline</slot>`
+            },
+            {
+              type: 'slot',
+              name: 'expandable',
+              value: `<slot slot='expandable'>Teaser's Expandable content</slot>`
+            },
+            {
+              type: 'template',
+              name: 'style',
+              value: '<div style="margin-bottom: 40px; width: 782px;">%TEMPLATE%</div>'
+            }
+          ]
+        });
+      })}
+    `;
+  }
+};
+
+export const Parts = {
+  parameters: {
+    controls: { exclude: ['base', 'media', 'content', 'meta', 'headline', 'main', 'expandable', 'variant'] }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'template',
+          name: 'sd-teaser-media::part(...){outline: solid 2px red}',
+          values: ['base', 'media', 'content', 'meta', 'headline', 'main', 'expandable'].map(part => {
```

Comment:
the `media` slot seems to be displayed/marked not completely. Also the margin below the teaser itself in the `base` slot compared to the `media` slot seems wrong?!?

Reply:
good point thanks! I adjusted the overflow behavior and forgot to check the impact on the border here in storybook